docs/source/models/cause.rst

```diff
@@ -558,6 +558,85 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+**Definition**
```

Comment:
Why not use another level of .rst subheadings instead of merely bolding the different subsection headings?

Reply:
For whatever reason when I was trying this the first time, they were all showing up the same regardless of what symbol I was using. Updating now, though.